webplugin/js/app/kommunicate-ui.js

```diff
@@ -3,6 +3,7 @@
  *
  */
 var kommunicateCommons = new KommunicateCommons();
+var GLOBAL = kommunicate._globals;
```

Comment:
KOMMUNICATE_GLOBAL is better name for this variable

Reply:
done